docs/src/Groups/grouphom.md

```diff
@@ -164,7 +159,13 @@ cokernel(f::GAPGroupHomomorphism)
 preimage(f::GAPGroupHomomorphism{S, T}, H::T) where S <: GAPGroup where T <: GAPGroup
 ```
 
-## Groups created by isomorphisms
+## Group isomorphisms
+
+```@docs
+isisomorphic(G::GAPGroup, H::GAPGroup)
+isomorphism(G::GAPGroup, H::GAPGroup)
+isisomorphic_with_isomorphism(G::Group, H::Group)
```

Comment:
`GAPGroup` instead of `Group`

Reply:
Thank you!

Which makes me realize that ideally we would eventually support this for non-GAP-groups, too... but it is unclear to me how we can do that in general... *sigh* But one stept after the other